lib/cb/responses/job/search_v3.rb

```diff
@@ -0,0 +1,28 @@
+module Cb
+  module Responses
+    module Job
+      class SearchV3 < ApiResponse
+
+        protected
+
+        def validate_api_hash
+          required_response_field(root_node, response)
+        end
+
+        def hash_containing_metadata
```

Comment:
I see that we have one of these today for v2. For my own information, what does this accomplish in v2 that we are now just going to give nil for?